package.json

```
@@ -1,169 +1,203 @@
+		"@babel/preset-env": "7.3.4",
+		"@babel/preset-react": "7.0.0",
+		"@babel/runtime": "7.3.4",
+		"@wordpress/api-fetch": "2.2.8",
+		"@wordpress/blob": "2.1.0",
+		"@wordpress/blocks": "6.0.7",
+		"@wordpress/browserslist-config": "2.3.0",
+		"@wordpress/components": "7.0.8",
+		"@wordpress/compose": "3.0.1",
+		"@wordpress/data": "4.2.1",
+		"@wordpress/date": "3.0.1",
+		"@wordpress/edit-post": "3.1.11",
+		"@wordpress/editor": "9.0.11",
+		"@wordpress/element": "2.1.9",
+		"@wordpress/hooks": "2.0.5",
+		"@wordpress/i18n": "3.1.1",
+		"@wordpress/keycodes": "2.0.6",
+		"@wordpress/plugins": "2.0.11",
+		"@wordpress/token-list": "1.1.0",
+		"@wordpress/url": "2.3.3",
+		"bounding-client-rect": "1.0.5",
+		"classnames": "2.2.6",
+		"click-outside": "2.0.2",
+		"clipboard": "2.0.4",
+		"component-uid": "0.0.2",
+		"cookie": "0.3.1",
+		"create-react-class": "15.6.3",
+		"cross-env": "5.2.0",
+		"css-loader": "2.1.1",
+		"del": "4.0.0",
+		"duplicate-package-checker-webpack-plugin": "3.0.0",
+		"email-validator": "2.0.4",
+		"es6-promise": "4.2.6",
+		"eslint": "5.12.0",
+		"eslint-config-prettier": "4.1.0",
+		"eslint-config-wpcalypso": "4.0.1",
+		"eslint-loader": "2.1.2",
+		"eslint-plugin-jsx-a11y": "6.1.2",
+		"eslint-plugin-lodash": "5.1.0",
+		"eslint-plugin-react": "7.11.1",
+		"eslint-plugin-wpcalypso": "4.0.2",
+		"fancy-log": "1.3.3",
+		"file-loader": "3.0.1",
+		"focus-trap": "4.0.2",
+		"gridicons": "3.1.1",
+		"gulp": "4.0.0",
+		"gulp-autoprefixer": "6.0.0",
+		"gulp-banner": "0.1.3",
+		"gulp-browserstack": "1.0.3",
+		"gulp-check": "0.1.2",
+		"gulp-clean-css": "2.4.0",
+		"gulp-concat": "2.6.1",
+		"gulp-if": "2.0.2",
+		"gulp-jshint": "2.1.0",
+		"gulp-json-transform": "0.4.6",
+		"gulp-minify": "3.1.0",
+		"gulp-modify": "0.1.1",
+		"gulp-po2json": "1.0.0",
+		"gulp-rename": "1.4.0",
+		"gulp-rtlcss": "1.3.0",
+		"gulp-sass": "4.0.2",
+		"gulp-sftp": "0.1.5",
+		"gulp-sourcemaps": "2.6.5",
+		"gulp-tap": "0.1.3",
+		"history": "3.3.0",
+		"i18n-calypso": "4.0.0-alpha.1",
+		"jsdom": "14.0.0",
+		"jsdom-global": "3.0.2",
+		"jshint": "2.9.7",
+		"jshint-stylish": "2.2.1",
+		"json-loader": "0.5.7",
+		"lodash": "4.17.11",
+		"mapbox-gl": "^0.53.1",
+		"markdown-it": "8.4.2",
+		"node-sass": "4.11.0",
+		"page": "1.7.1",
+		"photon": "2.0.1",
+		"plugin-error": "1.0.1",
+		"prop-types": "15.7.2",
+		"react": "16.8.4",
+		"react-dom": "16.8.4",
+		"react-pure-render": "1.0.2",
+		"react-redux": "6.0.1",
+		"react-router": "3.2.1",
+		"react-router-redux": "4.0.8",
+		"react-tagsinput": "3.19.0",
+		"redux": "4.0.1",
+		"redux-thunk": "2.3.0",
+		"refx": "3.1.1",
+		"resize-observer-polyfill": "1.5.1",
+		"sass-loader": "7.1.0",
+		"style-loader": "0.23.1",
+		"swiper": "4.4.6",
+		"uglify-save-license": "0.4.1",
+		"url-loader": "1.1.2",
+		"webpack": "4.29.6",
+		"whatwg-fetch": "1.1.1"
+	},
+	"devDependencies": {
+		"@automattic/calypso-build": "1.0.0-alpha.0",
+		"@automattic/calypso-color-schemes": "1.0.0-alpha.0",
```

This version is broken. We need `1.0.0-alpha.1` when https://github.com/Automattic/wp-calypso/pull/31628 lands and is published.

```suggestion
		"@automattic/calypso-color-schemes": "1.0.0-alpha.1",
```